webpack/components/Table/EmptyStateMessage.js

```diff
@@ -23,10 +26,22 @@ const KatelloEmptyStateIcon = ({
 };
 
 const EmptyStateMessage = ({
-  title, body, error, search, customIcon, happy,
+  title, body, error, search, customIcon, happy, ...extraTableProps
 }) => {
   let emptyStateTitle = title;
   let emptyStateBody = body;
+  let primaryActionTitle;
+  ({ primaryActionTitle } = extraTableProps);
+  let showPrimaryAction;
+  ({ showPrimaryAction } = extraTableProps);
+  let showSecondaryAction;
+  ({ showSecondaryAction } = extraTableProps);
+  let secondaryActionTitle;
+  ({ secondaryActionTitle } = extraTableProps);
+  let primaryActionLink;
+  ({ primaryActionLink } = extraTableProps);
+  let secondaryActionLink;
+  ({ secondaryActionLink } = extraTableProps);
```

Comment:
We never got around to going over this, but here's the simpler way to write it:

```js
const { primaryActionTitle, showPrimaryAction, showSecondaryAction,
secondaryActionTitle, primaryActionLink, secondaryActionLink } = extraTableProps;
```

see the "Basic assignment" section here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring